org.eclipse.lsp4e/src/org/eclipse/lsp4e/LSPEclipseUtils.java

```diff
@@ -228,13 +228,13 @@ public static CompletionParams toCompletionParams(URI fileUri, int offset, IDocu
 		if (document.getLength() > 0) {
 			try {
 				int positionCharacterOffset = offset > 0 ? offset-1 : offset;
-				String positionCharacter = document.get(positionCharacterOffset, 1);
-				if (Chars.contains(completionTriggerChars, positionCharacter.charAt(0))) {
-					param.setContext(new CompletionContext(CompletionTriggerKind.TriggerCharacter, positionCharacter));
+				String triggerCharacter = document.get(positionCharacterOffset, 1);
+				if (Chars.contains(completionTriggerChars, triggerCharacter.charAt(0))) {
+					param.setContext(new CompletionContext(CompletionTriggerKind.TriggerCharacter, triggerCharacter));
 				} else {
 					// According to LSP 3.17 specification: the triggerCharacter in CompletionContext is undefined if
 					// triggerKind != CompletionTriggerKind.TriggerCharacter
-					param.setContext(new CompletionContext(CompletionTriggerKind.Invoked, positionCharacter));
+					param.setContext(new CompletionContext(CompletionTriggerKind.Invoked, null));
```

Comment:
As I mentioned in https://github.com/eclipse/lsp4e/pull/747#discussion_r1296968755, I think we do not need to pass `null`, could we leave it out and use the one argument constructor?

Reply:
Ah, good point. Haven't seen that there is a one argument constructor.